fastlane/fastlane/Fastfile

```diff
@@ -112,13 +112,76 @@ lane :release do |options|
     )
   end
 
+  update_docs
+
   clubmate
 
   puts "You can now tweet:".green
   tag_url = "https://github.com/fastlane/fastlane/releases/tag/#{tool_name}/#{version}"
   puts "[#{tool_name}] #{github_release['name']} #{tag_url}"
 end
 
+desc "Makes sure the tests on https://docs.fastlane.tools still work with the latest version"
+lane :verify_docs do
+  local_fastlane_dir = File.expand_path("../..")
+  clone_docs do
+    temporary_gemfile = []
+    Fastlane::TOOLS.each { |tool| temporary_gemfile << "gem '#{tool}', path: '#{File.join(local_fastlane_dir, tool.to_s)}'" }
+    temporary_gemfile = temporary_gemfile.join("\n")
+    puts `sed -i -e "s/activate_bin_path/bin_path/g" $(which bundle)` # workaround for bundler https://github.com/bundler/bundler/issues/4602#issuecomment-233619696
+    sh "bundle install"
+    sh "bundle exec fastlane test skip_building:true" # skip_building since we don't have a proper python environment set up
+  end
+end
+
+desc "Update the actions.md on https://docs.fastlane.tools"
+desc "This will also automatically submit a pull request to fastlane/docs"
+lane :update_docs do
+  clone_docs do
+    require 'fastlane/documentation/markdown_docs_generator'
+    actions_md_path = File.expand_path("docs/actions.md")
+    Fastlane::MarkdownDocsGenerator.new.generate!(target_path: actions_md_path)
+
+    if `git status -s`.length == 0
```

Comment:
A minor suggestion, but I think `--porcelain` is the more common way to check this